rudder-core/src/main/scala/com/normation/rudder/services/servers/NewNodeManager.scala

```diff
@@ -458,6 +458,7 @@ trait ComposedNewNodeManager extends NewNodeManager with Loggable {
       case Full(seq) => //ok, cool
         logger.info(s"New node accepted and managed by Rudder: ${id.value}")
         cacheToClear.foreach { _.clearCache }
+        updateDynamicGroups.startManualUpdate
```

Comment:
this is pretty much adding back https://github.com/Normation/rudder/pull/986/files#diff-fb9876df49b189786ae8f6c12bb71679L454